handler/src/main/java/io/netty/handler/pcap/PCAPWriteHandler.java

```diff
@@ -99,31 +104,39 @@
 
     /**
      * Create new {@link PCAPWriteHandler} Instance.
-     * {@code captureZeroByte} are set to {@code false}.
+     * {@code captureZeroByte} is set to {@code false} and
+     * {@code writePcapGlobalHeader} is set to {@code true}.
      *
      * @param outputStream OutputStream where Pcap data will be written
      */
     public PCAPWriteHandler(OutputStream outputStream) {
-        this(outputStream, false);
+        this(outputStream, false, true);
     }
 
     /**
      * Create new {@link PCAPWriteHandler} Instance
      *
-     * @param outputStream    OutputStream where Pcap data will be written
-     * @param captureZeroByte {@code true} If we'll capture packets with 0 bytes
+     * @param outputStream          OutputStream where Pcap data will be written
+     * @param captureZeroByte       {@code true} If we'll capture packets with 0 bytes else {@code false}
+     * @param writePcapGlobalHeader {@code true} if we want to write Pcap Global Header on initialization
+     *                              of {@link PCapWriter} else {@code false}.
```

Comment:
The way these parameter docs are written makes the boolean parameters sound like a field that client code is reading, but the boolean values are more like commands that direct behaviour.

Here's an example of what it looks like when written in that style:

```
@param captureZeroByte Set to {@code true} to enable capturing packets with empty (0 bytes) payload. Otherwise, if {@code false}, empty packets will be filtered out.
```